osu.Game.Rulesets.Taiko.Tests/TaikoDifficultyCalculatorTest.cs

```diff
@@ -13,8 +13,8 @@ public class TaikoDifficultyCalculatorTest : DifficultyCalculatorTest
     {
         protected override string ResourceAssembly => "osu.Game.Rulesets.Taiko";
 
-        [TestCase(2.2867022617692685d, "diffcalc-test")]
-        [TestCase(2.2867022617692685d, "diffcalc-test-strong")]
+        [TestCase(2.2867022318057488d, "diffcalc-test")]
+        [TestCase(2.2867022318057488d, "diffcalc-test-strong")]
```

Comment:
Test adjustments like that in refactoring PRs should come with a proper rationale (which I've provided for you in another review comment).

Reply:
Fixed in 3743937774fb21887ee2472555274a5784d44048
see https://github.com/ppy/osu/pull/11689#discussion_r571817264